Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN apt-get update
 # hadolint ignore=DL3008
 RUN apt-get -y install --no-install-recommends rabbitmq-server
 RUN apt-get -y autoremove && apt-get -y clean
+ENV RABBITMQ_NODENAME=rabbit@localhost
+RUN rabbitmq-plugins enable rabbitmq_management
```

Comment:
Is this safe enough to do in production mode? Otherwise we'd need a condition again, something like:

```dockerfile
ARG DEBUG=0
RUN if [ "${DEBUG}" -gt 0 ]; then rabbitmq-plugins enable rabbitmq_management; fi;
```

Reply:
I think enabling the management plugin would not create any affect in terms of security, since we are exposing the management console only in debug mode.

wdyt?

Reply:
If enabling the plugin opens some internal ports with loose priviledges, then it should rather stay disabled, e.g. malicious workflow code could try to send there some commands, etc.  (Just trying to be cautious by design.) 

If enabling the plugin does not open anything by default, e.g. ecause one would have to add a piece of configuration,s then it may be OK to keep it here in both cases.

That said, since we are using the DEBUG technique for the other cluster images already, I guess it is safer to stick to it here as well for consistency?

```console
$ grep 'ARG DEBUG' r*/Dockerfile
reana-job-controller/Dockerfile:ARG DEBUG=0
reana-server/Dockerfile:ARG DEBUG=0
reana-workflow-controller/Dockerfile:ARG DEBUG=0
reana-workflow-engine-cwl/Dockerfile:ARG DEBUG=0
reana-workflow-engine-serial/Dockerfile:ARG DEBUG=0
reana-workflow-engine-yadage/Dockerfile:ARG DEBUG=0
```

Reply:
Another alternative is using [`RABBITMQ_ENABLED_PLUGINS_FILE`](https://www.rabbitmq.com/plugins.html#enabled-plugins-file) in the Helm chart. This way we'd avoid touching the Dockerfile.

Reply:
FWIW I'm seeing:

```console
$ reana-dev docker-build -c . --no-cache
...
Step 6/12 : RUN rabbitmq-plugins enable rabbitmq_management
 ---> Running in 71a130befb49
The following plugins have been enabled:
  mochiweb
  webmachine
  rabbitmq_web_dispatch
  amqp_client
  rabbitmq_management_agent
  rabbitmq_management

Applying plugin configuration to rabbit@localhost... failed.
 * Could not contact node rabbit@localhost.
   Changes will take effect at broker restart.
 * Options: --online  - fail if broker cannot be contacted.
            --offline - do not try to contact broker.
```

Reply:
This is for `prod` mode?